WordPress/Classes/ViewRelated/Site Creation/Web Address/DesignSystem/Foundation/Color+DesignSystem.swift

```diff
@@ -14,8 +14,15 @@ public extension Color {
             public static let error = Color(DesignSystemColorNames.Foreground.error)
 
             public static var brand: Color {
-                DS.Background.brand
+                if AppConfiguration.isJetpack {
+                    return jetpack
+                } else {
+                    return wordPress
+                }
```

Comment:
Is there a way to inject the "is Jetpack" flag instead of accessing the `AppConfiguration` singleton?

This kind of under the hood accessing of shared instances often ends up introducing coupling that then makes things harder to extract and modularize.

Reply:
Yeah you're certainly right. This isn't good at all for modularization. Would it makes sense that brand color is accessed as a function that accepts and argument?

Reply:
Yes.

I'm not sure where it would be best for the logic to switch to live. I guess DesignSystem is a good place because it's shared.